src/index.tsx

```diff
@@ -200,8 +209,36 @@ const Content: VFC<{ serverAPI: ServerAPI }> = ({}) => {
   );
 };
 
+function overrideTerminateApp() {
+  if (!(SteamClient.Apps as any)?.OriginalTerminateApp) {
+    console.log("Overriding SteamClient.Apps.TerminateApp(), original copy is saved to SteamClient.Apps.OriginalTerminateApp()");
+    (SteamClient.Apps as any).OriginalTerminateApp = SteamClient.Apps.TerminateApp.bind(SteamClient.Apps);
+  }
+
+  SteamClient.Apps.TerminateApp = (appid: number, unknownInput: any) => {
+    if (appid > 0xffffffff) {
+      // non-steam game
+      // https://gaming.stackexchange.com/questions/386882/how-do-i-find-the-appid-for-a-non-steam-game-on-steam
+      backend?.resumeApp?.(appid >> 32);
+    } else {
+      backend?.resumeApp?.(appid);
+    }
+    return (SteamClient.Apps as any).OriginalTerminateApp?.(appid, unknownInput);
+  };
+}
+
+function restoreTerminateApp() {
+  if ((SteamClient.Apps as any)?.OriginalTerminateApp) {
+    console.log("Restoring SteamClient.Apps.TerminateApp()");
+    SteamClient.Apps.TerminateApp = (SteamClient.Apps as any).OriginalTerminateApp;
+    delete (SteamClient.Apps as any).OriginalTerminateApp;
+  }
+}
+
```

Comment:
This crazy thing is for https://github.com/popsUlfr/SDH-PauseGames/issues/19
I bet the reviewers of the store won't like it so it's up to you... I'm checking on patching as a less intrusive solution but it's really making me scratch my head right now... May take some time before I can make something that works. The best would be that we can get some kind of an exception on patching the `SteamClient` object from them.

Reply:
Update: I got the approval from the reviewers in Discord, but I need to use the API provided by Decky Loader to do the override, I'll be looking into it.